### ISSUE TITLE
Adopt Bikeshed's dfn panel model

### ISSUE THREAD

**Issue by annevk** [whatwg/wattsi#117]
Bikeshed generates all the dfn panels as a long list of aside elements at the end of a document. I'm not sure if that is workable for HTML, but it might be interesting to try?

**Comment by domenic** [comment]
I don't think we need to adopt that model wholesale. We can just change the classnames of the generated element.

I guess that would mean modifications to https://github.com/whatwg/html/blob/master/html-dfn.js, instead.

**Comment by annevk** [comment]
It's a bit more complicated as HTML nests them inside the element so the styles end up with different requirements.
